pygmt/tests/test_clib_loading.py

```diff
@@ -131,7 +131,7 @@ def _mock_ctypes_cdll_return(self, libname):
         # libname is a loaded GMT library
         return self.loaded_libgmt
 
-    @pytest.fixture()
+    @pytest.fixture
```

Comment:
Note that this change is to fix a PT001 rule violation that was triggered because of the preview mode setting. In non-preview mode, `@pytest.fixture()` is valid, but in preview mode, `@pytest.fixture()` is valid. See https://docs.astral.sh/ruff/rules/pytest-fixture-incorrect-parentheses-style/ and https://github.com/astral-sh/ruff/pull/12106.